frontend/src/pages/Students.vue

```diff
@@ -0,0 +1,326 @@
+                              [
+                                  'tag is-success',
+                              ]">
+                          <li>Matched</li>
+                      </span> 
+
+                      <span v-if="props.row.StatusID == 3" :class="
+                              [
+                                  'tag is-danger',
+                              ]">
+                          <li>Dropped Out</li>
+                      </span> 
+
+                      <span v-if="props.row.StatusID == 4" :class="
+                              [
+                                  'tag is-warning',
+                              ]">
+                          <li>Unmatched</li>
+                      </span> 
+                  </template>
+
+
+
+                  </b-table-column>
+              </template>
+
+        
+
+            <!-- phone column --> 
+
+              <template v-for="(column, index) in columns" v-if="index == 4">
+                <b-table-column :key="column.id" v-bind="column" sortable>
+                  <template
+                    v-if="column.searchable"
+                    slot="searchable"
+                    slot-scope="props">
+                      <b-input
+                          v-model="props.filters[props.column.field]"
+                          icon="magnify"
+                          size="is-small" />
+                  </template>
+                  <template v-slot="props">
+                      {{ props.row.PhoneNumber }}
+                  </template>
+                  </b-table-column>
+              </template>
+              
+          </b-table>
+      </section>
+    </div>
+  </Page>      
+</template>
+
+
+<script>
+
+// Placeholder for API - Note that this will only work after the FullName method has been merged in the backend//
+const API_URL = "http://localhost:3001/api/students";
+
+import PageHeader from '../components/PageHeader.vue'
+import Page from '../components/Page.vue'
+
+
+export default {
+        data() {
+            return {
+                data: [
+                    //Placeholder data if you don't want to run the backend
+                    // { 'StudentID': 12345, 'FullName': 'Jesse Simmons', 'created_at': '2020-10-24T06:18:24.738Z', 'StatusID': 1,'PhoneNumber': '91233217' },
+                    // { 'StudentID': 23456, 'FullName': 'John Jacobs', 'created_at': '2020-10-25T06:18:24.738Z', 'StatusID': 2, 'PhoneNumber': '91312231' },
+                    // { 'StudentID': 31232, 'FullName': 'Tina Gilbert', 'created_at': '2020-10-26T06:18:24.738Z', 'StatusID':  3, 'PhoneNumber': '81234102'},
+                    // { 'StudentID': 41231, 'FullName': 'Clarence Flores', 'created_at': '2020-10-26T06:18:24.738Z', 'StatusID': 4,  'PhoneNumber': '93141234' },
+                    // { 'StudentID': 53212, 'FullName': 'Anne Lee', 'created_at': '2020-10-27T06:18:24.738Z', 'StatusID': 1,  'PhoneNumber': '81230532' },
```

Comment:
Let's remove all this commented code. The initial table data (before the backend call is done), should just be an empty array `[]`.